backend/tracim_backend/lib/core/content.py

```diff
@@ -1602,11 +1602,13 @@ def update_container_content(
          of content.
         :return:
         """
-        try:
-            item = self.set_allowed_content(item, allowed_content_type_slug_list)
-            content_has_changed = True
-        except SameValueError:
-            content_has_changed = False
+        content_has_changed = False
+        if allowed_content_type_slug_list is not None:
+            try:
+                item = self.set_allowed_content(item, allowed_content_type_slug_list)
+                content_has_changed = True
+            except SameValueError:
+                pass
```

Comment:
Could we log an error or something if we got this exception?

Reply:
I'm not sure that it is relevant. API client which, for example, resend `allowed_content_type_slug_list` any time (like current Tracim web client) will produce this error log each times.

Reply:
@Haskaris